client/src/components/Container/Container.js

```diff
@@ -10,12 +10,16 @@ const Container = ({ children, isAlt }) => {
     containerStyles = [styles.containerAlt];
   }
 
-  return <View style={containerStyles}>{children}</View>;
+  return (
+    <View style={containerStyles}>
+      <StatusBar barStyle="light-content" />
```

Comment:
hang on isn't `Container` used like all over the place? Unless I misunderstand `StatusBar` should probably only ever exist once at any point in time in the view hierarchy. I would have added it to the root application node in `app.js`. Maybe take a look at how some other react native apps do it?

Reply:
It is possible to have multiple StatusBar components mounted at the same time. The props will be merged in the order the StatusBar components were mounted.

Reply:
We can just remove - and address later - it was just for styling.